### ISSUE TITLE
ability to post GitHub output only on failures

### ISSUE THREAD

**Issue by LironEr** [LironEr/bundlemon#123]
> It would also be great to have a way to report the output differently in case of "failures".
For example, the report as PR comment can be very noisy in a monorepo with multiple projects reporting as PR comments, but if we can set the PR comment to be only in case of "failure", that would already be a lot less noisy.

_Originally posted by @tleunen in https://github.com/LironEr/bundlemon/issues/115#issuecomment-1216594565_

**Comment by LironEr** [comment]
Each output will accept
```
"always" / "on-failure" / "off"
```


default:
```json
"reportOutput": [
  [
    "github",
    {
      "checkRun": "off",
      "commitStatus": "always",
      "prComment": "always"
    }
  ]
]
```

@tleunen what do you think?

**Comment by tleunen** [comment]
Exactly what I had in mind, and you kept true/false in your PR to be backward compatible so this is great.
Thank you!

**Comment by LironEr** [comment]
Feature available in [bundlemon@2.0.0-rc.1](https://github.com/LironEr/bundlemon/releases/tag/bundlemon%402.0.0-rc.1)
